docs/api-v3-reference/openapi.yml

```
@@ -198,6 +197,97 @@ paths:
+        - in: query
+          name: version
+          required: false
+          schema:
+            type: string
+          description: Reserved for future version history. Currently returns `400 Bad Request` if provided.
+      responses:
+        "200":
+          description: Survey retrieved successfully
+          headers:
+            X-Request-Id:
+              schema: { type: string }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)